apps/docs/src/data/opencollective.data.ts

```
@@ -1,33 +1,16 @@
-export type CollectiveMembersResponse = {
-  MemberId: number
-  createdAt: string
-  type: string
-  role: string
-  tier: string
-  isActive: boolean
-  totalAmountDonated: number
-  currency: string
-  lastTransactionAt: string
-  lastTransactionAmount: number
-  profile: string
-  name: string
-  company: null | null
-  description: string | null
-  image: string | null
-  email: null | string
-  twitter: null | string
-  github: string | null
-  website: null | string
-}
+import {type CollectivePartialResponse} from '../types'
 
 const collectiveSlug = 'bootstrap-vue-next'
 const openCollectiveBaseURL = 'https://opencollective.com'
 const openCollectiveMembersFetchUrl = `${openCollectiveBaseURL}/${collectiveSlug}/members/all.json`
 
+// This is a stubbed export to satisfy typescript, for some reason
```

Also see my comment on the PR - would be delighted if there is a better solution for this - but I can't seem to get typescript to recognise this without declaring it or to suppress the error.

---

suppress the error

---

> suppress the error

Thanks. For some reason, I was blocking on the fact that the error didn't have an error number associated with it - but I used ts-expect-error for the line; that way, if the issue gets fixed downstream, this should start failing, and we can pull the suppression.